src/extensions/timeseries/TimeWidgetNew/mainLayer.ts

```diff
@@ -13,6 +13,7 @@ interface LayerState {
     inverseTransformations: TransformationMatrix[]
     anchorTimepoint: number | null
     dragging: boolean
+    captureWheel: boolean
```

Comment:
Tracks whether the `mainLayer` element has focus, and thus should scale with mouse wheel events, or doesn't, and thus should pass them through (but doesn't).